### ISSUE TITLE
Game tries to update then tells me to restart (which does nothing). 

### ISSUE THREAD

**Issue by Beardedcannuck** [dalaranwow/dalaran-wow#4232]
[//]: # (INFORMATION ABOUT TAGS) title basically says it all. I am not sure how to fix it. Is there a patch somewhere that I am missing? 
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by AlphaTrion0** [comment]
I am also having the same issue.  I've even uninstalled the client server and reinstalled them and I am getting the same problem.

**Comment by Rushor** [comment]
https://www.reddit.com/r/DalaranWoW/comments/3cmrkn/stuck_on_downloading_update/

1.) use the correct realmlist: set realmlist logon.dalaran-realmlist.org
2.) Make shure, that the wow.exe and the wow filder has not "write protection"
3.) Use accountname (not e-mail) and you passwort from the website for the ingame login.

The issuetracker is serverside and gamemechanic issues only.

/closed
